test/core/modules1/add-qr.js

```diff
@@ -0,0 +1,4 @@
+const testModule = require('../templates/module-test.1'),
+  options = {size:200, qrCodeString:'https://github.com/publiclab/image-sequencer'};
+console.log('############ WASM Benchmarks ############');
```

Comment:
Hmm, actually perhaps this line doesn't belong in the `add-qr` module; could you move it to `module-test.1` or somewhere outside the modules?

Reply:
Ya I am looking into it!!